src/lib/annotations/Annotator.js

```diff
@@ -836,13 +771,23 @@ class Annotator extends EventEmitter {
      * Unbinds event listeners for annotation modes.
      *
      * @protected
+     * @param {string} mode - Annotation mode to be unbound
      * @return {void}
      */
-    unbindModeListeners() {
+    unbindModeListeners(mode) {
         while (this.annotationModeHandlers.length > 0) {
```

Comment:
This part delegates the unbinding to the controller rather than the annotator. Meaning once a point annotation controller is in (and preferably highlight as well) this call just needs to be a call to controller[mode].unbindModeListeners()